apiritif/loadgen.py

```diff
@@ -176,9 +176,12 @@ def run_nose(self, params):
             config.stream = open(os.devnull, "w")  # FIXME: use "with", allow writing to file/log
         try:
             while True:
+                iteration += 1
```

Comment:
does this mean iterations start with 1 and not from 0?

Reply:
This is intentional. I believe that iterations should be counted from 1, not from zero. But I'm open for suggestions.